src/com/jetbrains/RoundedCornersManager.java

```diff
@@ -46,6 +46,14 @@ public interface RoundedCornersManager {
      *   <li>"small" - round the corners if appropriate, with a small radius</li>
      * </ul>
      *
+     * Possible values for Linux ({@link java.lang.String}):
```

Comment:
Maybe mention that this is only for Wayland, not Linux in general?

Reply:
Certainly! Forgot to push the latest version where this was preemptively resolved.